### PR TITLE
Support test case numbers

### PR DESCRIPTION
**Pull request by NozomiIto**
(and small refactoring)

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -219,6 +219,12 @@ inputs:
         - "true"
         - "false"
       is_required: true
+  - test_case_numbers: ""
+    opts:
+      title: "Test Casd Numbers"
```
Comment (nkns165) [inline step.yml]: There's typo `Casd`.
And I think "Test case numbers" is better, as other parameter names.

**Review by nkns165 — CHANGES_REQUESTED** [review]
Sorry to be late, reviewed

**Review by nkns165 — APPROVED** [review]

**Comment by NozomiIto** [comment]
I have not checked the UI on the Bitrise UI yet due to Bitrise trouble or something...

**Comment by nkns165** [comment]
😢

**Comment by NozomiIto** [comment]
https://github.com/bitrise-io/bitrise-steplib/pull/2407

**Pull request merged**